src/main/kotlin/me/serce/solidity/ide/formatting/SolFormattingBlock.kt

```diff
@@ -41,9 +42,16 @@ class SolFormattingBlock(
       blocks.add(e)
       child = child.treeNext
     }
+    if (astNode.treeParent == null) {
+      logger<SolFormattingBlock>().warn("blocks: ${blocks.print(0)}")
```

Comment:
Was this logger line left here accidentally, or is it necessary for something? If accidental, it's worth removing to avoid spamming logs on user devices.

Reply:
removed this debug code. Thanks!